pages/common/img2pdf.md

```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF.
+> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Convert images to PDF:
+
+`img2pdf {{path/to/image1.jpg}} {{path/to/image2.jpg}} --output {{path/to/file.pdf}}`
+
+- Convert all PNGs in the current directory to a single pdf:
+
+`img2pdf {{*.png}} --output {{path/to/file.pdf}}`
+
+- Convert only the first page of multi-page images to pdf:
+
+`img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
+
+- Auto orient to A4 page size in landscape mode and a border of 2cm horizontally and 5.1cm vertically:
```

Comment:
> 
Perhaps this will read better?
```suggestion
- Auto orient image, use a page size of A4 in landscape mode, and set a border of 2cm horizontally and 5.1cm vertically:
```